Sources/Extensions/Foundation/FileManagerExtensions.swift

```diff
@@ -56,6 +56,22 @@ public extension FileManager {
         return nil
     }
 
+    /// SwifterSwift: Decode Encodable object from file.
+    ///
+    /// - Parameters:
+    ///   - type: type of the object that is being decoded.
+    ///   - decoder: JSONDecoder.
+    ///   - url: URL a file with saved object data.
+    /// - Throws: Throws any errors thrown by JSONDecoder.
+    /// - Returns: An object of specified type. Returns nil if failed to read from URL.
+    public func decodeJSON<T: Decodable>(as type: T.Type, using decoder: JSONDecoder = JSONDecoder(), from url: URL) throws -> T? {
+        guard let data = contents(atPath: url.path) else {
```

Comment:
Put this all in 1 line
```suggestion
        guard let data = contents(atPath: url.path) else { return nil }
```